tests/mocha/block_test.js

```diff
@@ -49,6 +49,29 @@ suite('Blocks', function() {
     delete Blockly.Blocks['statement_block'];
   });
 
+  function createTestBlocks(workspace, isRow) {
```

Comment:
I like the helper method. If all the tests use this.blocks could we just set that in the parent setup method?

Reply:
I tried setting this.blocks, but it seems like the "this" in the function is not the same "this" as in the test suites.

Reply:
I think I meant just having this.block = createTestBlocks() in the parent setup function.